TAC/Meetings/2025/2025-04-03/CCC-TAC-Minutes-2025-04-03.md

```diff
@@ -0,0 +1,149 @@
+Paul Howard(PH) mentioned that the Certifier Framework project community has requested assistance in securing funding for an internship opportunity. They have a technical proposal involving a research student from the University of Missouri. The request is for funding from the Confidential Computing Consortium (CCC) to support this mentored internship.PH, being new to the Certifier Framework project and unfamiliar with the CCC funding application process, sought guidance on how to proceed in his liaison role to support this request.
+DM acknowledged the interest and the identification of a project and student. DM suggested that the CCC governance repository contains information on the funding process, although it might appear complex. He recommended that Paul Howard contact Yash Mankad or Riaan, who have more direct knowledge of this process.
+
+* ISO Liaison: Alec Fernandez (AF) informed the group about a project proposal at ISO to create a document defining confidential computing. He added that the draft document is titled "Definitions or an introduction to confidential computing." Due to copyright restrictions, the draft cannot be circulated at this time. AF mentioned that he is initiating the formal process of establishing a liaison with ISO. The goal is to determine the appropriate entity (Linux Foundation, Joint Standards Foundation, or Confidential Computing Consortium) to hold the liaison. He also added that he is consulting with legal counsel at the Linux Foundation for guidance on the correct approach. He highlighted that the potential ways to engage with ISO include: 
+Submitting a white paper for approval as an ISO standard. 
+Collaborating with an existing ISO working group.
+AF noted that the current ISO draft uses definitions that are primarily SGX-centric and cites work that is also heavily focused on SGX. He aims to broaden these definitions to include other confidential computing technologies like TDX and those from AMD and Arm.
+
+* Security Baseline introduction
+Emily Fox (EF) presented an introduction to the Security Baseline. The presentation generated a significant number of questions from attendees & were effectively addressed by EF. DM requested a unification of best practices, badges, and OSPS (Open Source Project Security )baseline. EF acknowledged this concern and stated it would be a priority for her team to reconcile the best practices badging system, especially as passing criteria is a requirement for many LF Foundation projects.
```

Comment:
```suggestion
Emily Fox (EF) presented an introduction to the Security Baseline. The presentation generated a significant number of questions from attendees & were effectively addressed by EF. DM requested a unification of best practices, badges, and OSPS (Open Source Project Security) baseline. EF acknowledged this concern and stated it would be a priority for her team to reconcile the best practices badging system, especially as passing criteria is a requirement for many LF Foundation projects.
```